examples/simpleReadEvery/main.go

```diff
@@ -0,0 +1,64 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/xeoncross/wav"
```

Comment:
hmmm :)

don't know how you work with go project contributions yet but this works better for me until I make the decision to make a real fork, in the sense of changing it from the original, not contributing back:

1. make a fork on github
2. `go get`/clone the origin into your `$GOPATH`
3. add your forked remote to the source repo


like this:

```
$ cd $GOPATH/src/github.com/cryptix/wav
$ git remote add fork git@github.com:xenocross/wav.git
$ git add changes..
$ git commit -m 'awesome stuff'
$ git push fork
```

than you can make a pull request from your fork to mine without having to deal with to instances of the packge in your gopath.